cirq-core/cirq/circuits/optimization_pass.py

```diff
@@ -36,22 +36,21 @@ def __init__(
         new_operations: 'cirq.OP_TREE',
         preserve_moments: bool = False,
     ) -> None:
-        """
-        Args:
-            clear_span: Defines the range of moments to affect. Specifically,
-                refers to the indices in range(start, start+clear_span) where
-                start is an index known from surrounding context.
-            clear_qubits: Defines the set of qubits that should be cleared
-                with each affected moment.
-            new_operations: The operations to replace the cleared out
-                operations with.
-            preserve_moments: If set, `cirq.Moment` instances within
-                `new_operations` will be preserved exactly. Normally the
-                operations would be repacked to fit better into the
-                target space, which may move them between moments.
-                Please be advised that a PointOptimizer consuming this
-                summary will flatten operations no matter what,
-                see https://github.com/quantumlib/Cirq/issues/2406.
+        """Args:
```

Comment:
First line should be a description of the method. In this case I think the args should be moved to `Attributes` on the class. https://google.github.io/styleguide/pyguide.html#384-classes